app/instance-initializers/head.js

```diff
@@ -1,30 +1,33 @@
 import ENV from '../config/environment';
 
 export function initialize(owner) {
-  if (ENV['ember-cli-head'] && ENV['ember-cli-head']['suppressBrowserRender']) { return true; }
+  if (typeof FastBoot === 'undefined') {
+    if (ENV['ember-cli-head'] && ENV['ember-cli-head']['suppressBrowserRender']) { return true; }
 
-  // clear fast booted head (if any)
-  let startMeta = document.querySelector('meta[name="ember-cli-head-start"]')
-  let endMeta = document.querySelector('meta[name="ember-cli-head-end"]')
-  if (startMeta && endMeta) {
-    let el = startMeta.nextSibling
-    while(el && el !== endMeta) {
-      document.head.removeChild(el);
-      el = startMeta.nextSibling;
+    // clear fast booted head (if any)
+    let startMeta = document.querySelector('meta[name="ember-cli-head-start"]')
+    let endMeta = document.querySelector('meta[name="ember-cli-head-end"]')
+    if (startMeta && endMeta) {
+      let el = startMeta.nextSibling
+      while(el && el !== endMeta) {
+        document.head.removeChild(el);
+        el = startMeta.nextSibling;
+      }
+      document.head.removeChild(startMeta);
+      document.head.removeChild(endMeta);
     }
-    document.head.removeChild(startMeta);
-    document.head.removeChild(endMeta);
-  }
 
-  const component = owner.lookup('component:head-layout');
-  component.appendTo(document.head);
+    const component = owner.lookup('component:head-layout');
+    component.appendTo(document.head);
+  } else {
```

Comment:
> I might be misunderstanding things, but by adding a file to the fastboot manifest, aren't you creating a file that is loaded only in fastboot?

@scottmessinger yes. The usecase that you are talking about earlier is if you have to import external libraries in fastboot environment. The usecase in this PR is different than that are refers to case 4 in the cheatsheet. 

> Outside the scope of discussion for this repo but the churn on Fastboot 1.0 is very difficult to keep up on.

@bcardarella Have you seen the migration cheatsheet? Would love your feedback on the issue Scott posted with any additional usecases that seem confusing for you and we can work with making things better. Any help to make it better is greatly appreciated. 

Things will be much simpler to understand once the website is updated.